cfgov/v1/models/molecules.py

```diff
@@ -117,16 +117,76 @@ class TextIntroduction(blocks.StructBlock):
 
     class Meta:
         icon = 'title'
-        template = 'v1/demo/molecules/text_introduction.html'
+        template = '_includes/molecules/text-introduction.html'
+
+
+class Hero(blocks.StructBlock):
+    heading = blocks.CharBlock(max_length=100, required=True)
+    body = blocks.RichTextBlock(required=False)
+
+    image = ImageChooserBlock(required=False)
+    image_path = blocks.CharBlock(required=False)
+    image_alt = blocks.CharBlock(required=False)
+
+    background_color = blocks.CharBlock(max_length=100, required=False)
```

Comment:
Not sure if this is something we should be handling in the CMS or not. @sebworks @anselmbradford @jimmynotjim @KimberlyMunoz how is background determined in the front end? If there's a manageable set of background colors to use then we can set up a ChoiceField to select from that list if it makes sense to do so. What do yall think?